docs/azure/ai/get-started-app-chat-template.md

```diff
@@ -0,0 +1,300 @@
+
+1. Open **Visual Studio Code** in the context of an empty directory.
+
+1. Ensure that you have the [Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) installed in Visual Studio Code.
+
+1. Open a new terminal in the editor.
+
+    > [!TIP]
+    > You can use the main menu to navigate to the **Terminal** menu option and then select the **New Terminal** option.
+    >
+    > :::image type="content" source="../media/get-started-app-chat-template/open-terminal-option.png" lightbox="../media/get-started-app-chat-template/open-terminal-option.png" alt-text="Screenshot of the menu option to open a new terminal.":::
+
+1. Sign in to Azure with the Azure Developer CLI.
+
+    ```bash
+    azd auth login
+    ```
+
+    When prompted, copy the code from the terminal and then paste it into a browser. Follow the instructions to authenticate with your Azure account.
+
+1. Initialize the folder to use the sample project with Azure Developer CLI:
+
+    ```bash
+    azd init -t azure-search-openai-demo-csharp
+    ```
+
+    You don't need to clone this repository.
+
+1. Open the **Command Palette**, search for the **Dev Containers** commands, and then select **Dev Containers: Reopen in Container**.
+
+    > [!TIP]
+    > Visual Studio Code may automatically prompt you to reopen the existing folder within a development container. This is functionally equivalent to using the command palette to reopen the current workspace in a container.
+
+1. The remaining exercises in this project take place in the context of this development container.
+
+---
+
+## Deploy and run
+
+The sample repository contains all the code and configuration files you need to deploy a chat app to Azure. The following steps walk you through the process of deploying the sample to Azure.
+
+### Deploy chat app to Azure
+
+> [!IMPORTANT]
+> Azure resources created in this section immediate costs, primarily from the Cognitive Search resource. These resources may accrue costs even if you interrupt the command before it is fully executed.
+
+1. Run the following Azure Developer CLI command to provision the Azure resources and deploy the source code:
+
+    ```bash
+    azd up
+    ```
+
+1. When you're prompted for an environment name, enter a value of your choice, such as `dotnetai`. The environment name influences the naming of provisioned Azure resources.
+1. When you're prompted to select a location the first time, select a location near you. This location is used for most the resources including hosting.
+1. When you're prompted for a location for the OpenAI model, select a location that is near you. If the same location is available as your first location, select that.
+1. Wait until app is deployed. It may take 5-10 minutes for the deployment to complete.
```

Comment:
I thought that there were wait times up to 20 mins, have you tested this? @bradygaster do you recall how long it would take?

Reply:
You're correct about this